### PR TITLE
content-healthcheck script

### PR DESCRIPTION
**Pull request by lenuhc**
* added script for getting a descriptor file from an s3 bucket.
* parse the content paths
* hit their endpoints in the paired AEM publisher instance
* send a cloudwatch health status custom metric for each one of the endpoints

### CODE REVIEW

File: templates/aem-tools/content-healthcheck.py.epp [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    for tags in ec2instance.tags:
+        if tags["Key"] == tagKey:
+            tagValue = tags["Value"]
+    return tagValue
+
+# Downloads descriptor file from an s3 bucket into a local tmp folder
+def downloadFileFromS3():
+    bucket_key = '<%= $stack_prefix %>/content-healthcheck-descriptor.json'
+    s3 = boto3.client('s3')
+    s3.download_file(<%= $data_bucket %>, bucket_key, local_path)
```
Comment (lenuhc) [inline templates/aem-tools/content-healthcheck.py.epp]: Not sure if <%= $data_bucket %> needs to be inside quotes

File: templates/aem-tools/content-healthcheck.py.epp [hunk 2]
```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```
Comment (cliffano) [inline templates/aem-tools/content-healthcheck.py.epp]: Just adding a note that we still have to improve consistency across various python code (logging mechanism, args parsing, etc). That will be done as part of https://github.com/shinesolutions/aem-aws-stack-provisioner/issues/6 .

File: templates/aem-tools/content-healthcheck.py.epp (hunk 1 above)
Comment (cliffano) [inline templates/aem-tools/content-healthcheck.py.epp]: Yes it has to.

File: templates/aem-tools/content-healthcheck.py.epp [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import urllib, json, requests, urllib3, boto3, sys, argparse
+
+local_path = '<%= $tmp_dir %>/content-healthcheck-descriptor.json'
+region = 'ap-southeast-2'
```
Comment (cliffano) [inline templates/aem-tools/content-healthcheck.py.epp]: correction, region should be taken care of by the instance...

**Pull request merged**